training_system/config/locales/en.yml

```diff
@@ -1,5 +1,9 @@
 en:
-  hello: "Hello world"
+  activerecord:
+    models: 
+      course:
+        end_date: "End Date"
+        status: "Status"
```

Comment:
Em không thấy anh edit vi.yml ạ, thiếu đúng k a

Reply:
không có vi.yml e =))))

Reply:
Ủa a không làm vi luôn hả. Em cứ tưởng phải làm chứ, trước làm project em toàn làm đủ, thật thà quá mà =))